package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aria-query",
-  "version": "5.2.0",
+  "version": "5.2.1",
```

Comment:
hopefully this commit will be ff-merged onto master and tagged appropriately :-)